packages/css-engine/src/core/rules.ts

```diff
@@ -11,6 +11,9 @@ class StylePropertyMap {
   constructor(transformValue?: TransformValue) {
     this.#transformValue = transformValue;
   }
+  setTransformer(transformValue: TransformValue) {
```

Comment:
do we have tests for transformers in css-engine? if not please add a few, its important to keep this core piece well-tested

Reply:
Yes, we have